sru-scripts/manual/softlayer-sru

```diff
@@ -32,11 +32,12 @@ create_setup_proposed_script
  
 echo "### BEGIN $UBUNTU_RELEASE"
 VM_IP=`launch-softlayer --image=os:$UBUNTU_RELEASE -u $SAMPLE_CLOUDCONFIG | awk '/primary ip/{printf "root@%s", $NF}'`
-sleep 180
+sleep 100
```

Comment:
Reduced this count as we now run retry_ssh_command() which handles remaining sleeps